client/dashboard/app/analytics/index.tsx

```diff
@@ -22,3 +24,17 @@ export function AnalyticsProvider( { children, client }: AnalyticsProviderProps
 export function useAnalytics() {
 	return useContext( AnalyticsContext );
 }
+
+export function bumpStat( group: string, name: string ) {
```

Comment:
The code for recording bump stats is so small I just included it inline rather than trying to share it from anywhere.

You can see the docs for it here

PCYsg-J0-p2